perun-core/src/main/java/cz/metacentrum/perun/core/impl/Utils.java

```diff
@@ -522,9 +520,6 @@ public static int getNewId(JdbcTemplate jdbc, String sequenceName) {
 			case "postgresql":
 				query = "select nextval('" + sequenceName + "')";
 				break;
-			case "hsqldb":
```

Comment:
Again, we can probably stop distinguish DBs, unless this method is useful in some ExtSource-like logic.

Reply:
Fixed